modules/default/weather/providers/weatherbit.js

```diff
@@ -95,7 +95,7 @@ WeatherProvider.register("weatherbit", {
 	// Implement WeatherDay generator.
 	generateWeatherDayFromCurrentWeather (currentWeatherData) {
 		//Calculate TZ Offset and invert to convert Sunrise/Sunset times to Local
-		const d = new Date();
+		const d = Date.now();
 		let tzOffset = d.getTimezoneOffset();
```

Comment:
Since `Date.now() returns the number of milliseconds since January 1, 1970.` methods like `getTimezoneOffset` will not work with your change.

Reply:
or those in calendarUtils

Reply:
Oops. Fixed up in latest push